Source/Magick.NET/Shared/Profiles/Exif/ExifDefinition.cs

```diff
@@ -0,0 +1,364 @@
+﻿using System.Collections;
+
+namespace ImageMagick
+{
+    /// <summary>
+    /// Class that can be used to provide a definition of ExifValues within an EXIF profile.
+    /// </summary>
+    public class ExifDefinition
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExifDefinition"/> class.
+        /// </summary>
+        /// <param name="exifTag"></param>
+        /// <param name="exifDataType"></param>
+        /// <param name="isArray"></param>
+        internal ExifDefinition(ExifTag exifTag, ExifDataType exifDataType, bool isArray)
+        {
+            ExifTag = exifTag;
+            ExifDataType = exifDataType;
+            IsArray = isArray;
+        }
+
+        /// <summary>
+        /// Gets a value indicating ExifTag for the current definition.
+        /// </summary>
+        public ExifTag ExifTag
+        {
+            get; 
+            private set;
```

Comment:
You don't need to add `private set` when you only set the property in the constructor.

Reply:
True, if the ExifDefinition class becomes internal. But I'd like it to be public, in which case private set is necessary for a read only property.